tests/common/fixtures/snappi_fixtures.py

```diff
@@ -0,0 +1,413 @@
+"""
+This module contains the snappi fixture
+"""
+import pytest
+import snappi
+from ipaddress import ip_address, IPv4Address
+from tests.common.fixtures.conn_graph_facts import conn_graph_facts,\
+    fanout_graph_facts
+from tests.common.ixia.common_helpers import get_vlan_subnet, get_addrs_in_subnet,\
```

Comment:
Can you create a new folder test.common.snappi to hold all the helper functions used to snappi?

Reply:
created new folder and moved helper files as suggested